shaders/src/common_material.fs

```diff
@@ -81,13 +81,18 @@ struct MaterialInputs {
     vec3  subsurfaceColor;
 #endif
 
-#if defined(SHADING_MODEL_CLOTH) || defined(SHADING_MODEL_SPECULAR_GLOSSINESS)
+#if defined(SHADING_MODEL_CLOTH)
     vec3  sheenColor;
 #if defined(MATERIAL_HAS_SUBSURFACE_COLOR)
     vec3  subsurfaceColor;
 #endif
 #endif
 
+#if defined(SHADING_MODEL_SPECULAR_GLOSSINESS)
+    vec3  specularColor;
```

Comment:
You need to remove `roughness` when the model is specular glossiness

Reply:
Done